src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.1.2">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid11.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
```

Comment:
Please, remove this MonoAndroid11 support

Reply:
I moved to Android 11's ADK because it has different API to set status bar style. If I revert this change than I would need to revert support for Android 11 API for changing it.

It should still work on Android 11 (I'll check that), but this code will need to be added later when XCT will move to Android 11+.

Reply:
Yeah, but we need to discuss internally when we will move to support Android11 and that will be a hard job, because we will need to test everything and see if it works. So for now I prefer to drop the Android 11 support

Reply:
@pictos Removed Android 11 support checked the code on Android 11 device. It doesn't work. If we cannot update XCT to Android 11 SDK, we cannot implement this feature (who would use it if it doesn't work on latest OS version?). 
I think we should be fine updating to Android 11 SDK (everything should still work).

Reply:
For history: here is the commit needed to support Android 11's API https://github.com/xamarin/XamarinCommunityToolkit/pull/812/commits/38e872a1c54c4956ef0bc62edbb423059eb5d426